docs/sql/grammar.md

```diff
@@ -181,3 +171,102 @@ to `GROUP BY`.
 
 `MINUS` is equivalent to `EXCEPT`.
 
+### Window aggregates
+
+One type of expression that can appear in a `SELECT` statement is a
+window aggregate.  The grammar for window aggregates is:
+
+```
+windowedAggregateCall
+  : agg '(' [ ALL | DISTINCT ] value [, value ]* ')'
+      [ RESPECT NULLS | IGNORE NULLS ]
+      [ WITHIN GROUP '(' ORDER BY orderItem [, orderItem ]* ')' ]
+      [ FILTER '(' WHERE condition ')' ]
+      OVER windowSpec
+  | agg '(' '*' ')'
+      [ FILTER  '(' WHERE condition ')' ]
+      OVER windowSpec
+
+windowSpec
+  :   '('
+      [ windowName ]
+      PARTITION BY expression [, expression ]*
+      [ ORDER BY orderItem [, orderItem ]* ]
+      [
+          RANGE BETWEEN windowRange AND windowRange
+      ]
+      ')'
+
+windowRange
+  :   CURRENT
+  |   ( UNBOUNDED | expression ) ( PRECEDING | FOLLOWING )
+```
+
+where `agg` is one of the operators in the following table.
+
+<table>
```

Comment:
We don't support all of these yet, do we?

Reply:
The grammar does, but they are marked in the wishlist as unsupported